Taiwan-Ebook-Lover/Taiwan-Ebook-Lover.github.io
# Update workflow

## Summary



Update the workflow to ensure proper generation of Release Notes and deployment.
Adjust the timing of checking for [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).

## How did you test this change?



- Upgrade husky
- Check commit message match Conventional Commits rules before commit code, instead of checking PR title
- Update release workflow
